apps/bluetooth/bluetooth-service.js

```diff
@@ -213,7 +213,7 @@ function onConnectionStateChangedListener (mode, state, device) {
       break
     case protocol.CONNECTION_STATE.DISCONNECTED:
       if (lastUrl !== '/close' && lastUrl !== '/implied_close') {
-        speak(getText('DISCONNECTED'))
+        speak(getText('DISCONNECTED'), res.AUDIO[state])
```

Comment:
AFAIK, `getText('DISCONNECTED')` and `res.AUDIO[state]` should share the parameter `'DISCONNECTED'`, could we?

Reply:
> AFAIK, `getText('DISCONNECTED')` and `res.AUDIO[state]` should share the parameter `'DISCONNECTED'`, could we?

Rewrite speak logic, bind replacement audio url to string content.